x/gov/types/v1/proposal.go

```diff
@@ -23,7 +23,7 @@ const (
 )
 
 // NewProposal creates a new Proposal instance
-func NewProposal(messages []sdk.Msg, id uint64, metadata string, submitTime, depositEndTime time.Time, title, summary string, proposer sdk.AccAddress) (Proposal, error) {
+func NewProposal(messages []sdk.Msg, id uint64, submitTime, depositEndTime time.Time, metadata, title, summary string, proposer sdk.AccAddress, expedited bool) (Proposal, error) {
```

Comment:
@tac0turtle Should we group metadata, title and summary next to each other too in v0.47?
I feel like it makes more sense. Otherwise, metadata is a bit _alone_ and the position is a bit weird.

Reply:
Done in https://github.com/cosmos/cosmos-sdk/pull/14782